app/experimenter/nimbus-ui/src/components/PageEditAudience/index.test.tsx

```diff
@@ -55,12 +73,36 @@ describe("PageEditAudience", () => {
     render(<Subject mocks={[mock, mutationMock]} />);
     await screen.findByTestId("PageEditAudience");
     fireEvent.click(screen.getByTestId("next"));
+    expect(experiment.status).toEqual(NimbusExperimentStatusEnum.DRAFT);
     await waitFor(() => {
       expect(mockSubmit).toHaveBeenCalled();
       expect(navigate).toHaveBeenCalledWith("../");
     });
   });
 
+  it("handles form next button for live updates", async () => {
+    render(<Subject mocks={[mockRollout, rolloutMutationMock]} />);
```

Comment:
So the `mockRollout` here is already in a live state eh?  I wish the setups here were a little more explicit.  I don't just mean this specific test, I just mean the frontend tests in general.  I like to see a setup where the exact conditions necessary are explicitly declared.  Particularly since we're asserting on the status later.  Not a specific direction, just a thought about the frontend tests in general.

Reply:
I see what you mean. Maybe something to chat about next time the three of us get together?